doc/cprover-manual/goto-analyzer.md

```diff
@@ -59,7 +59,7 @@ goto-analyzer --show --dot depgraph.dot --dependence-graph-vs
 
 
 
-### Task {#task}
+### <a name="task"></a> Task
```

Comment:
The `name` attribute has been deprecated in HTML5. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-name

`id` should be used instead, although the `id` is generally auto-generated (from the heading content) by both GitHub's markdown viewer, and all major Markdown -> HTML converters.

Reply:
Thank you for the clarification and also making me take a second look at what is being rendered right now (which has broken links). You are absolutely right that `id` is auto-generated, it was just being confused by the extra part in braces (we ended up with `id="tasktask"`).